src/allmydata/test/cli/wormholetesting.py

```diff
@@ -32,14 +32,15 @@ async def local_peerB(helper, mw):
 
 from __future__ import annotations
 
-from typing import Iterator, Optional, List, Tuple
-from collections.abc import Awaitable
-from inspect import getargspec
+__all__ = ['IWormhole']
```

Comment:
Now the module says that the *only* name it exports is `IWormhole` ... but it should still claim to export all of the rest.

In other words, `__all__` is the complete export list.  It is not additive with the default export list.

Reply:
Okay I added the needed elements to complete the export list.